etc/scripts/docenizerJava.sh

```diff
@@ -0,0 +1,5 @@
+#/usr/bin/bash
+
+curl https://docs.oracle.com/javase/specs/jvms/se16/html/jvms-6.html -o `pwd`/vendor/jvms.html
+
+node docenizer-java.js > ../../lib/handlers/asm-docs-java.js
```

Comment:
I'm wondering if using the `find-node.sh` script to get the node path would work better here, but I have no idea

Reply:
Updated in 86b09bee67c6897be260ccd45cfe46e8c543536c